website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md

```diff
@@ -0,0 +1,145 @@
+4. Copy the displayed token and click Done. (don’t lose it!)
+
+
+For your development credentials/profiles.yml:
+
+1. Set your default catalog to dev.
+2. Your developer schema should be named after yourself. We recommend dbt_<first_name_initial><last_name>.
+
+By doing this, dbt will create the developer schemas if it doesn’t already exist in the **dev** catalog but can be overridden to specify sources or production targets.
+
+### Defining your dbt deployment environment
+
+Last, we need to give dbt a way to deploy code outside of development environments. To do so, we’ll use dbt [environments](https://docs.getdbt.com/docs/collaborate/environments) to define the production targets that end users will interact with.
+
+Core projects can use [targets in profiles](https://docs.getdbt.com/docs/get-started/connection-profiles#understanding-targets-in-profiles) to separate environments. [dbt Cloud environments](https://docs.getdbt.com/docs/get-started/develop-in-the-cloud#set-up-and-access-the-cloud-ide) allow you to define environments via the UI and [schedule jobs](https://docs.getdbt.com/docs/get-started/getting-started/building-your-first-project/schedule-a-job) for specific environments.
+
+Let’s set up our deployment environment:
+
+1. In Databricks, [set up your service principal’s token](https://docs.databricks.com/dev-tools/service-principals.html#use-curl-or-postman). To do this, you will need to open up your terminal.
+2. In your terminal, define 2 environment variables:
+    1. **DATABRICKS_HOST** - the url used to log into your Databricks workspace
+    2. **DATABRICKS_TOKEN** - your [personal access token](https://docs.databricks.com/dev-tools/auth.html#pat) (also used for dbt Cloud developer credentials)
+
+```bash
+export DATABRICKS_HOST="https://dbc-a1b2345c-d6e78.cloud.databricks.com"
+export DATABRICKS_TOKEN="dapi1234567890b2cd34ef5a67bc8de90fa12b"
+```
+
+3. Next, run a curl command to get the “application_id” of your service principal. You can also get this value from the Databricks workspace admin console > Service Principals > Edit
+
+```bash
+curl -X GET \
+${DATABRICKS_HOST}/api/2.0/preview/scim/v2/ServicePrincipals \
+--header "Authorization: Bearer ${DATABRICKS_TOKEN}" \
+| jq .
+```
+
+4. Then you create a JSON file with the application ID retrieved from the above command, a comment to describe what the token will be used for, and the amount of time the token will be valid (in seconds) before it expires and will need to be regenerated.
+
+```bash
+vim create-service-principal.json
+```
+
+```json
+{
+  "application_id": "<application-id>",
+  "comment": "<comment>",
+  "lifetime_seconds": 1209600
+}
+```
+
+5. Run one more curl command to get the service principal’s token:
+
+```bash
+curl -X POST \
+${DATABRICKS_HOST}/api/2.0/token-management/on-behalf-of/tokens \
+--header "Content-type: application/json" \
+--header "Authorization: Bearer ${DATABRICKS_TOKEN}" \
+--data @create-service-principal-token.json \
+| jq .
```

Comment:
Again we can't assume that `jq` exists

Reply:
This is straight out of the Databricks doc linked directly above it. There are details in there about alternatives. I think reproducing everything in the Databricks doc would make our guide hard to read (like the Databricks docs)

Reply:
>There are details in there about alternatives

Any code we provide is going to be copy-pasted and run immediately without any review, and for most people it's going to fail. The fact that there are instructions on a different website saying "by the way this won't work immediately" doesn't solve that IMO. 

Since this is a one-off process, I would make a pretty strong case for our section of the documentation being based on the Postman instructions, not the CLI (assuming that there's no UI - see above comment). Then we could even provide a Postman example that people could run themselves, right?

Reply:
Removed these steps since there are too many edge cases to consider